### DISCUSSION TITLE
Tips for better storms.

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#79] [Show and tell]
This is more for new people, but old ones may take something into account too. So here i will tell some tips for good storms from my expirience. Need to note: i've been using this simulator since first webgl release, so i have quite a lot of knowledge. 

Let's start with things you don't want to do:
Don't make vorticity too high, or too low. High vorticity can make updrafts break up easily, while low can make clouds look to cartoonish, and more like lava lamp wax. Also high vorticity with time can lead to unwanted unrealistic turbulence. Default vorticity should do it job. 

If you want to simulate enviroment by itself, but not storm than i would recommend not adding phantom wind with wind option. Because if you want to simulate enviroment by itself, let it just naturally appear.

Do not turn off Global drying, while it can improve storms and even add new cloud types to simulation, with time simulation space will slowly fill with clouds as humidity will not have exit.

Don't set sun intensity bigger than 1.5 . Because sky will be so bright that weather stations will not be visible, and temperature will rise so quickly that humidity will not renew quick enough and you will have your sim die out.

Don't make lake/sea temperature at max, it will make so many heat that humidity will not renew quick enough.

Don't make evaporation heat high. Reason same as 2 previous.

Don't make water weight lower than default. Then humidity will be easy target for updrafts, and well... same as 3 previous.

Don't make terrain too hilly. Too much turbulence will break up updrafts.

What you should do for good storms:

Combined terrain: have good hybrid of flat, water, mount terrain, where most is land, and least mounts for good storm development.

Wide map area: wider maps usually can hold more terrain features, and overall more weather events can happen.

Humidity and heat balance: try keeping heating less powerful than humidity emit rate, so you will have humidity build up with time.

Time: just wait, sometimes it may take 5 simulation days to get great land storm!

Watch out: look out for cold fronts, high temperature groups, ect. Usually best storms are short living, and form at cold fronts, ect. So make sure not to skip them by accident.

And last, wait: If you see weak convections, don't look away thay may lead into more powerful storms later!

**Comment by alexlew822008-boop** [comment]
tornado

**Comment by jackborlok-create** [comment]
tornado

**Comment by kennethranile-png** [comment]
What is this
